### PR TITLE
Events list widget styles

### PR DESCRIPTION
**Pull request by paulmskim**
**Ticket:** [TEC-3596]

This PR covers the FE styles for the events list widget.

**Artifact:**

image
image
image

[TEC-3596]: https://moderntribe.atlassian.net/browse/TEC-3596

### CODE REVIEW

File: src/views/v2/widgets/events-list.php [hunk 1]
```diff
@@ -73,26 +73,34 @@
...
-			</h3>
-		</header>
+	<div class="tribe-common-l-container">
+		<div class="tribe-events-widget-events-list">
+			<header class="tribe-events-widget-events-list__header">
+				<h3 class="tribe-events-widget-events-list__header-title tribe-common-h6 tribe-common-h--alt">
+					<?php
+					echo esc_html(
+						sprintf(
+							_x( 'Upcoming %1$s', 'Title for events list widget.', 'the-events-calendar' ),
```
Comment (tr1b0t) [inline src/views/v2/widgets/events-list.php:83]: :no_entry_sign: A gettext call containing placeholders was found, but was not accompanied by a "translators:" comment on the line above to clarify the meaning of the placeholders.

<sub>hash: 54540900282e5c3a5e155813f20f4e88</sub>

File: src/views/v2/widgets/events-list.php [hunk 2]
```diff
@@ -73,26 +73,34 @@
...
-				<?php $this->template( 'widgets/events-list/event', [ 'event' => $event, 'display' => $display ] ); ?>
-			<?php endforeach; ?>
-		<?php else : ?>
-			<?php // get messages component ?>
-		<?php endif; ?>
+			<?php if ( ! empty( $events ) ) : ?>
+
+				<div class="tribe-events-widget-events-list__events">
+					<?php foreach ( $events as $event ) : ?>
+						<?php $this->template( 'widgets/events-list/event', [ 'event' => $event, 'display' => $display ] ); ?>
```
Comment (tr1b0t) [inline src/views/v2/widgets/events-list.php:95]: :no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 4827e98bc198be025a156684b1f57109</sub>

File: src/views/v2/widgets/events-list.php [hunk 3]
```diff
@@ -73,26 +73,34 @@
...
+					<?php endforeach; ?>
+				</div>
+
+			<?php else : ?>
+
+				<?php // get messages component ?>
```
Comment (tr1b0t) [inline src/views/v2/widgets/events-list.php:101]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 7032da56b23800606ac48af4a06afa0d</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 3 error(s)

**Comment by paulmskim** [comment]
on hold until https://github.com/moderntribe/the-events-calendar/pull/3299 is merged. code review can happen, but do not merge.

**Review by lucatume — COMMENTED** [review]
Probably requires a snapshot update.

File: src/Tribe/Views/V2/Assets.php [hunk 4]
```diff
@@ -99,6 +99,49 @@ public function register() {
 			]
 		);
 
+		/**
+		 * @todo: @paulmskim figure out how to enqueue only when widget is available.
+		 * @todo: @paulmskim commented out for release so widget styles do not enqueue, uncomment when actually releasing.
+		 */
+		// tribe_asset(
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:106]: :warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 27f994313dad8231b64be0067d90d4a8</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 5]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	$plugin,
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:107]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: bcb0686daa7c33c5bf4d7828ed016af8</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 6]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	'tribe-events-widgets-v2-skeleton',
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:108]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 6de0a2cebd6918606b440512767a72f1</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 7]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	'widgets-skeleton.css',
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:109]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 33c95dc71e4925259b2f3ab0e7206990</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 8]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	[
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:110]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 3d924d0de3685a32171fd7e340d3cc1e</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 9]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 		'tribe-common-skeleton-style',
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:111]: :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: 77cda82b56afedc134e9cd9fcbe32582</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 10]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	],
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:112]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 9ecddc647d705c468d76b98e7ddd45bb</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 11]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	'wp_enqueue_scripts',
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:113]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: ce6ee744fe70aa5d4c4a1d6994a46748</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 12]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	[
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:114]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 32b505ec346bf31ff0898f0b293a783a</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 13]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 		'priority'     => 10,
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:115]: :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: 61b9359efd992b747f8b1e3ef57cbbad</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 14]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 		'conditionals' => [ $this, 'should_enqueue_frontend' ],
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:116]: :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: 4920c2c66f6a0b23796756dacecedade</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 15]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 		'groups'       => [ static::$group_key ],
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:117]: :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: c337dcc2b984ee99693d838b89117756</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 16]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	]
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:118]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 061c02d2fee354eda52477c10ee8e1ae</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 17]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// );
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:119]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 84852c49874ec934fcf9f87ca25108c6</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 18]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+
+		/**
+		 * @todo: @paulmskim figure out how to enqueue only when widget is available.
+		 * @todo: @paulmskim commented out for release so widget styles do not enqueue, uncomment when actually releasing.
+		 */
+		// tribe_asset(
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:125]: :warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 8aa9dbf4c60427a36fd28876dba5f0da</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 19]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	$plugin,
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:126]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: b6f881e2242de298e5f8e6a98e74aa3f</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 20]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	'tribe-events-widgets-v2-full',
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:127]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 005eef9204bdb78d16cdfdce7e4507d7</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 21]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	'widgets-full.css',
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:128]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: e41a8cb5f0ff353037b4532e6f344387</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 22]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	[
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:129]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 3b62dbd35ee1d10c9a4de126cb26bcf1</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 23]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 		'tribe-common-full-style',
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:130]: :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: bf79fb42d44cc0f15e848eb7d561f6d0</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 24]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 		'tribe-events-widgets-v2-skeleton',
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:131]: :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: f2ef9067c8f548bf7f36d41fe724718f</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 25]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	],
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:132]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 72736908855d8ca9215392f173e23f20</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 26]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	'wp_enqueue_scripts',
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:133]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 4a4c8bea34688542844a948a5b8cb032</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 27]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	[
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:134]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: d66b4c30049646c17bf202d6c7f3c70c</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 28]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 		'priority'     => 10,
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:135]: :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: fce8f47b6b082f8da81f10f69fbf33ea</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 29]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 		'conditionals' => [
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:136]: :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: 7bc0e245ea54aab8f779109a98a6366b</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 30]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 			'operator' => 'AND',
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:137]: :no_entry_sign: Expected 1 space before comment text but found 10; use block comment if you need indentation

<sub>hash: d24422a3f7dfccdbd34bb13f1dada648</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 31]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 			[ $this, 'should_enqueue_frontend' ],
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:138]: :no_entry_sign: Expected 1 space before comment text but found 10; use block comment if you need indentation

<sub>hash: e4cb3b0acb547629ff9e3dfb5efb22bb</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 32]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 			[ $this, 'should_enqueue_full_styles' ],
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:139]: :no_entry_sign: Expected 1 space before comment text but found 10; use block comment if you need indentation

<sub>hash: c967f76617daebcebf4ab68b90d6b272</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 33]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 		],
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:140]: :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: f4a3d08517e5c14f4212eaa136ce13c6</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 34]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 		'groups'       => [ static::$group_key ],
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:141]: :no_entry_sign: Expected 1 space before comment text but found 6; use block comment if you need indentation

<sub>hash: e90a968009ae486bcf6837d407183b9c</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 35]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// 	]
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:142]: :no_entry_sign: Expected 1 space before comment text but found 2; use block comment if you need indentation

<sub>hash: 67d49bd7ce8b360e8b2cad523e4f8e97</sub>

File: src/Tribe/Views/V2/Assets.php [hunk 36]
```diff
@@ -99,6 +99,49 @@ public function register() {
...
+		// );
```
Comment (tr1b0t) [inline src/Tribe/Views/V2/Assets.php:143]: :no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: a86de9d0c5fa684c83ad0e94cd49b57a</sub>

**Review by tr1b0t — COMMENTED** [review]
:no_entry_sign: 34 error(s)
:warning: 2 warning(s)

File: src/views/v2/widgets/events-list.php (hunk 3 above)
Comment (paulmskim) [reply src/views/v2/widgets/events-list.php:102]: Will remove later.

File: src/views/v2/widgets/events-list.php (hunk 2 above)
Comment (paulmskim) [reply src/views/v2/widgets/events-list.php:96]: Fixed in a later PR.

**Review by bordoni — APPROVED** [review]

**Pull request merged**